### DISCUSSION TITLE
Is 'SEED' limited to only one IP address ?

### DISCUSSION THREAD

**Discussion by ahmedjami** [stargate/stargate#1891] [Help] [answered]
Hi,
It seems that `SEED` should be configured with only one IP address or when I tested it with an existing Cassandra 3 nodes clusters and stoped the `SEED` node, stargate fails and stops connection to cassandra and I see the follwoing error:

The stargate conf:
```
...................
- JAVA_OPTS="-Xmx2G"
- CLUSTER_NAME=c4-${CASSTAG}-cluster
- CLUSTER_VERSION=4.0
- SEED=backend-1
- RACK_NAME=rack1
- DATACENTER_NAME=datacenter1
................................
```
The exception that I see when I stop` backend-1`
```
INFO [Messaging-EventLoop-3-1] 2022-06-15 13:47:04,059 NoSpamLogger.java:92 - /172.22.0.5:7000-> backend-1/172.22.0.2:7000-URGENT_MESSAGES-[no-channel] failed to connect
io.netty.channel.ConnectTimeoutException: connection times out: backend-1/172.22.0.2:7000
```

This is not a SPOF for stargate? Or I'm missing something to configure stargate correctly?

Greetings,

**Comment by EricBorczuk** [comment]
Hi @ahmedjami,

Would you mind sharing the complete `starctl` command you are running (with any sensitive information removed)?

**Comment by ahmedjami** [comment]
Hi @EricBorczuk 
Thanks for your reply.
I'm using the docker-compose that you are providing in stargate repo. Here is the link: https://github.com/stargate/docker-images/tree/master/examples/cassandra-4.0
As you can see from this file, docker-compose will start 3 Cassandra nodes and stargate that is configured to use backend-1 as a seed `CASSANDRA_SEEDS=backend-1`.

**Comment by jeffreyscarpenter** [comment, marked as answer]
You can tell Stargate to use multiple seeds:

```
  stargate:
  ...
    environment:
      - SEED=backend-1,backend-2
```
